### ISSUE TITLE
Not running with TF 2

### ISSUE THREAD

**Issue by jvence** [javaidnabi31/Multi-Label-Text-classification-Using-BERT#8]
Is this suppose to run with TF2? Tried running it in a virtual env on Mac with no success. Any help would be greatly appreciated.

**Comment by ghost** [comment]
Nope, bert-tensorflow is build for tensorflow 1.x execution
For bert tf2 refer to [https://github.com/kpe/bert-for-tf2](url)

**Comment by jvence** [comment]
Any plans of making work with TF2?

**Comment by ghost** [comment]
bert-tensorflow is based on [https://github.com/google-research/bert](url), not compatible with tf2, if you want to implement bert in tensorflow 2, there are two ways

1. Use the individual codes from google-bert and migrate it to tf2
2.Use [https://github.com/kpe/bert-for-tf2](url). This is based on google-bert implementation and made compatible for tf2. The code could be imported through pip install bert-for-tf2, an example for its implementation- [https://www.kaggle.com/jugglingsnakeboarder/realornotnlp-bert-use-svm](url)

I would suggest the use of bert-for-tf2, because its a Keras layer(It also utilizes kpe/params-flow to reduce common Keras boilerplate code ) implementation of the bert, a better utilization of tensorflow 2 potential
